core/src/main/java/org/ehcache/core/EhcacheManager.java

```diff
@@ -679,7 +679,7 @@ public void destroyCache(final String alias) throws CachePersistenceException {
     }
 
     try {
-      removeAndCloseWithoutNotice(alias);
+      removeCache(alias);
```

Comment:
oh, you mean ... we don't already test destroyCache() ? ;-)
I'll have a look. I'll also get a very deep breath for daggy fixing...